### PR TITLE
Editorial: call out examples more clearly + add explainer

### LINKED ISSUE #59: Add explainer to the spec repo
Even though this spec is fairly old, we need an explainer for the TAG review (#43, w3ctag/design-reviews#529).

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #59

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/pull/65.html" title="Last updated on Mar 23, 2021, 10:03 PM UTC (7575f84)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/geolocation-api/65/c0381e2...7575f84.html" title="Last updated on Mar 23, 2021, 10:03 PM UTC (7575f84)">Diff</a>

### CODE REVIEW

**Comment by xfq** [comment]
Thanks. I think it would be useful to mention that no location information is made available through this API without the user's permission.

**Comment by marcoscaceres** [comment]
I sprinkled a few "If the user allows it, " where permission is asked. I'll add a few more.

**Comment by marcoscaceres** [comment]
I added an additional, "Once a user grants permission, ..."

**Comment by marcoscaceres** [comment]
Plus added "no location information is made available through this API without the user's permission" after the listing of method.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,52 @@
...
+
+## Examples of usage
+
+This HTTPs-only API exposes the `navigator.geolocation` object with a couple of useful methods:
+
+- `.getCurrentPosition(successCallback, [errorCallback, options])` - "one shot" position request
+- `.watchPosition(successCallback, [errorCallback, options])` - Watch a position and get notified of any changes.
+- `.clearWatch(someId)` - allows you to stop watching for location changes.
+
+No location information is made available through this API without the user's permission
```
Comment (xfq) [inline README.md:19]:
```suggestion
No location information is made available through this API without the user's permission.
```

**Review by xfq — APPROVED** [review]
LGTM

**Pull request merged**